tests/unit/test_rds_utils.py

```diff
@@ -149,6 +178,10 @@ def test_is_reader_cluster_dns(test_value):
     china_region_cluster,
     china_region_instance,
     china_region_proxy,
+    china_region_custom_domain,
+    china_region_cluster,
+    china_region_instance,
+    china_region_proxy,
     china_region_custom_domain
 ])
 def test_is_not_reader_cluster_dns(test_value):
```

Comment:
Should we add tests for the other patterns that were added (elb, gov dns/cluster patterns)?